src/main/java/com/sparta/spangeats/domain/store/service/StoreService.java

```diff
@@ -1,4 +1,50 @@
 package com.sparta.spangeats.domain.store.service;
 
+import com.sparta.spangeats.domain.member.entity.Member;
+import com.sparta.spangeats.domain.member.enums.MemberRole;
+import com.sparta.spangeats.domain.store.dto.StoreRequestDto;
+import com.sparta.spangeats.domain.store.dto.StoreResponseDto;
+import com.sparta.spangeats.domain.store.entity.Store;
+import com.sparta.spangeats.domain.store.exception.StoreLimitExceededException;
+import com.sparta.spangeats.domain.store.exception.AccessDeniedException;
+import com.sparta.spangeats.domain.store.exception.StoreNameAlreadyExistsException;
+import com.sparta.spangeats.domain.store.repository.StoreRepository;
+import jakarta.transaction.Transactional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+
+@Service
 public class StoreService {
+
+    private final StoreRepository storeRepository;
+    @Autowired
+    public StoreService(StoreRepository storeRepository) {
+        this.storeRepository = storeRepository;
+    }
```

Comment:
위의 코드랑 통일성을 맞추면 좋을 거 같습니다~~ 모두 `@RequiredArgsContructor`를 써주거나 아니면 하나하나 생성자를 만들어주는 식으로요!

```suggestion
@Service
@RequiredArgsContructor
public class StoreService {

    private final StoreRepository storeRepository;
    }
```